scripts/ci/libraries/_local_mounts.sh

```diff
@@ -44,6 +44,7 @@ function local_mounts::generate_local_mounts_list {
         "$prefix"hooks:/opt/airflow/hooks:cached
         "$prefix"logs:/root/airflow/logs:cached
         "$prefix"pylintrc:/opt/airflow/pylintrc:cached
+        "$prefix"pylintrc:/opt/airflow/pylintrc:cached
```

Comment:
Dupe here too

Reply:
Ah that would explain one of the failures. By accident I have not saved the pyproject.toml change here and it was generated from here to the dockerfile :). It's needed for black formatting of the generated files.